webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+
+module.exports = {
+  entry: __dirname + '/src/main.jsx',
+  output: {
+    path: __dirname + '/build/js',
+    filename: '[name].bundle.js'
+  },
+  externals: {
+    'react': 'React',
```

Comment:
I'm extremely unfamiliar w/ the context here but it might be to [improve cross-site performance by using a shared CDN](https://github.com/mozilla/teach.webmaker.org/issues/817) for common assets? :bowtie: 

(That is, since all our sites use React, it might be beneficial for us to load it from a common CDN, so that e.g. a user's browser doesn't load React twice when the user hops between teach.m.o and id.wm.o.)